lib/routes/charge/pos_invoice.dart

```diff
@@ -233,31 +233,59 @@ class POSInvoiceState extends State<POSInvoice> {
                                                 child: Align(
                                                     alignment:
                                                         Alignment.centerLeft,
-                                                    child: BadgeIcon(
+                                                    child: GestureDetector(
+                                                      onTap: () {
```

Comment:
Clicking the shopping cart does not open the  SaleView half the time. Add `behavior: HitTestBehavior.translucent,` parameter to GestureDetector so that BadgeIcon allow it to receive events.